### PR TITLE
New package: syncthingtray-1.2.2

### PR DESCRIPTION
**Pull request by classabbyamp**
- New package: qtforkawesome-0.0.4
    - dependency
- New package: syncthingtray-1.2.2

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/syncthingtray/template [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+if [ -n "$XBPS_CHECK_PKGS" ]; then
+	makedepends+=" libcppunit-devel"
+fi
+
+pre_check() {
+	export SYNCTHING_PORT=$(comm -23 <(seq 49152 65535) <(ss -tan | awk '{print $4}' | cut -d':' -f2 | grep "[0-9]\{1,5\}" | sort | uniq) | shuf | head -n 1)
+}
+
+subpackages="syncthingtray-devel $(vopt_if kde 'syncthingtray-plasma syncthingtray-dolphin')"
```
Comment (ahesford) [inline srcpkgs/syncthingtray/template:47]: I'd move this up top with the other metadata.

File: srcpkgs/syncthingtray/template [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+maintainer="classabbyamp <void@placeviolette.net>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/Martchus/syncthingtray"
+distfiles="https://github.com/Martchus/syncthingtray/archive/refs/tags/v${version}.tar.gz"
+checksum=0f556e30091ffc06e7e489c5e09e5460401c28fd1e8318b68cb6066ead7fe7a1
+
+make_check_pre="env SYNCTHING_TEST_TIMEOUT_FACTOR=3"
+make_check_target="check"
+
+build_options="kde webengine"
```
Comment (ahesford) [inline srcpkgs/syncthingtray/template:27]: I'd be inclined to say the KDE option should be omitted and always enabled until somebody presents a pressing need to disable it. (Of course, I'm also the one who added some gratuitous qt options to tageditor...)

File: srcpkgs/syncthingtray/template [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		build_options_default+=" webengine"
+	fi
+fi
+
+if [ -n "$XBPS_CHECK_PKGS" ]; then
+	makedepends+=" libcppunit-devel"
+fi
+
+pre_check() {
+	export SYNCTHING_PORT=$(comm -23 <(seq 49152 65535) <(ss -tan | awk '{print $4}' | cut -d':' -f2 | grep "[0-9]\{1,5\}" | sort | uniq) | shuf | head -n 1)
```
Comment (ahesford) [inline srcpkgs/syncthingtray/template:44]: O_O

Is this even worth doing?

Comment (classabbyamp) [reply srcpkgs/syncthingtray/template:44]: I grabbed it from the AUR package (maintained by the author). it just finds a free port in that range for tests to use.

Comment (ahesford) [reply srcpkgs/syncthingtray/template:44]: Yeah, I'm just wondering if it's worth doing this dance to find a port, or if checks should just be disabled.

Comment (classabbyamp) [reply srcpkgs/syncthingtray/template:44]: it's a small number of tests, so I don't see any issue with keeping it

**Comment by classabbyamp** [comment]
everything else should be addressed now

**Pull request merged**